lib/course_meetings_manager.rb

```diff
@@ -97,7 +97,7 @@ def calculate_week_meeting_dates
     meetings = []
     @timeline_week_count.times do |wk|
       week_start = @beginning_of_first_week + wk.weeks
-      week_end = week_start.end_of_week(:saturday)
+      week_end = week_start.end_of_week(:sunday)
```

Comment:
Probably ought to have a comment explaining that this code is supposed to return the last day, Saturday, of the week.

Reply:
Can do. I actually think just `.end_of_week` will work as well. Perhaps that's better syntax anyhow?

Reply:
Yeah, `end_of_week` default is probably good, but maybe merits a comment anyway, since the intention remains to return a saturday but it's not obvious that's what will happen.

Reply:
`end_of_week` _does_ work but if you call it on a Sunday it just picks the same day. Added a comment and left in the `:sunday` keyword.